charts/portal/README.md

```diff
@@ -460,9 +460,6 @@ If the RabbitMQ nodes are stopped or removed out of order, there is a chance tha
 $ helm upgrade <release-name> --set-file <values-from-install> --set <values-from-install> -f <my-values.yaml> layer7/portal
 ```
 
-## Disclaimer
-This repository is currently in Beta.
-
 ## License
 Copyright (c) 2019 CA, A Broadcom Company. All rights reserved.
```

Comment:
Isn't the Copyright to be 2020 instead of 2019?

Reply:
Updated